test/integration/install.js

```diff
@@ -35,6 +36,11 @@ describe('install', function () {
       install(__dirname, {_: ['', name + '@' + version]}, function (err) {
         assert.ifError(err)
         require(name)
+        assert(pathIsInside(require.resolve(name), __dirname), 'should resolve package from the target directory')
```

Comment:
Added here an extra check that we have got `test/integration/node_modules/mocha`, but not `node_modules/mocha` from dependencies of `ied` itself.